packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -52,10 +52,10 @@ abstract class Route<T> {
   @mustCallSuper
   void install(OverlayEntry insertionPoint) { }
 
-  /// Called after install() when the route is pushed onto the navigator.
+  /// Called after install() when the route is pushed onto the navigator. The
```

Comment:
I'd add a paragraph break after the first sentence.

The first line of the dartdoc becomes the summary for the method that is displayed on the class page on the web site.